docs/config/rules.md

```diff
@@ -63,17 +63,17 @@ the corresponding CSS will be generated:
 .p-5 { padding: 1.25rem; }
 ```
 
-Congratulations! Now you got your own powerful atomic CSS utilities. Enjoy!
+Congratulations! Now you've got your own powerful atomic CSS utilities. Enjoy!
 
-## Full controlled rules
+## Fully controlled rules
 
-::: warning
-This is an advanced feature, you don't need it in most of the cases.
+::: tip
+This is an advanced feature, it can be safely ignored
```

Comment:
I am not sure of this one, it's not about "safety" or "ignore" this as this is not a "warning" or "problem" - in that way, I agree with your "warning->tip" change 😄